### PR TITLE
Math: Updates to Random generator tests

### PR DESCRIPTION
**Pull request by marcusnaslund**
Part 5 of #491 
Possible conflict with #516 but should be easy to fix in that case. Either can go first.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Resolved conflicts.

File: test/math/IntRandomGeneratorTest.ooc [hunk 1]
```diff
@@ -51,6 +52,23 @@ IntRandomGeneratorTest: class extends Fixture {
 			expect(Int absolute(mean - expectedMean) < 2)
 			expect(Int absolute(deviation - expectedDeviation) < 2)
 		})
+		this add("uniform range", func {
+			uniformGenerator := IntUniformRandomGenerator new()
+			uniformGenerator setRange(-250_000, 250_000)
+			uniformLowest := 250_000
+			uniformHighest := -250_000
+			for (i in 0 .. 100_000) {
```
Comment (davidhesselbom) [inline test/math/IntRandomGeneratorTest.ooc]: I just learned about the `_` variable from @tomasmahlberg:
```ooc
for (_ in 0 .. 100_000) {
   // code that doesn't actually use `i` for anything
]
```
But I'm not going to ask you to change this now.

Comment (marcusnaslund) [inline test/math/IntRandomGeneratorTest.ooc]: Like in Erlang. Good to know.

Comment (davidhesselbom) [inline test/math/IntRandomGeneratorTest.ooc]: Shame you can't do `for (0 .. 100_000)` though.

**Pull request merged**